src/components/card/card.jsx

```diff
@@ -12,11 +12,23 @@ import * as styles from "./card.module.scss";
 const Card = ({ data }) => {
   const categoryBlog = useCategory(data.frontmatter.category);
 
+  const urlImages = () => {
+    if (typeof window !== 'undefined' && `${window.location.href}/` === window.location.origin) {
+      return data.frontmatter.thumbnail;
+    } else {
+      if (typeof window !== 'undefined') {
+        const imageUrl = window?.location?.origin;
+        return `${imageUrl}/${data.frontmatter.thumbnail}`
```

Comment:
What happens if window.location or window.location.origin don't exist? is the ur the same as the previous return?

Reply:
Can I write window.location.origin || data.frontmatter.thumbnail?